core/primitives/src/version.rs

```diff
@@ -172,7 +186,9 @@ impl ProtocolFeature {
             #[cfg(feature = "protocol_feature_block_header_v3")]
             ProtocolFeature::BlockHeaderV3 => 109,
             #[cfg(feature = "protocol_feature_chunk_only_producers")]
-            ProtocolFeature::ChunkOnlyProducers => 115,
+            ProtocolFeature::ChunkOnlyProducers => 126,
```

Comment:
can we do that (jumping feature from one protocol version to another?)

or do we have to do a fork of the nightly chain anyways?

Reply:
We fork betanet with each release